mods/digimon/pokedex.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
```

Comment:
Suggestion for the whole file:

inherit from exisitng pokemon that look simaler to the digimon for sprites.